packages/template-retail-react-app/package.json

```diff
@@ -93,7 +93,7 @@
     },
     {
       "path": "build/vendor.js",
-      "maxSize": "330 kB"
+      "maxSize": "350 kB"
```

Comment:
The vendor has increased about 20k and it's because of chakra libs

```
@chakra-ui/theme (+118 KB)
@chakra-ui/react/modal-focus (+91 KB)
@chakra-ui/skip-nav (+79 KB)
@chakra-ui/react/use-popper (+77 KB)
@chakra-ui/styled-system/system (+33 KB)
@chakra-ui/react/use-number-input (+18 KB)
@chakra-ui/react/popover (+17 KB)
@chakra-ui/react/chakra-provider (+15 KB)
@chakra-ui/react/checkbox (+15 KB)
@chakra-ui/react/modal (+12 KB)
```

Reply:
Oh no! Do we want to pin chakra-ui on older version? The size penalty doesn't seem to justify the gain, I don't know what benefit are we getting in the new versions...